weatherbot/get_temp.py

```diff
@@ -14,4 +12,4 @@ def conv_kelv_to_cels(temp):
     return round(cels)
 
 def add_C(cels):
-    return (cels + "°C").encode('utf-8')
+    return (str(cels) + "°C").encode('utf-8')
```

Comment:
Just in case, what you really want here is `return str(cels) + u"\xb0C"`

Reply:
Thank you, I fixed it.